<launch>

	<arg name="robot_name" default="trina2_1"/>
	<node pkg="pick_and_place" name="trina_grasps" output="screen" type="grasp_generator" ns="$(arg robot_name)">
	</node>
</launch>